### PR TITLE
Bah 8940 supplemental benefits

### PR DESCRIPTION
**Pull request by imeecuison**
## Description
Supplemental benefits form

## Testing done
tests/pages/supplementalBenefits.unit.spec.jsx

## Screenshots
screen shot 2018-12-10 at 9 44 19 am

## Acceptance criteria
- [ ] The user is able to populate the following:
Are you receiving, or do you expect to receive, disability retirement benefits? {radio}

Yes

No
- [ ] Are you receiving, or do you expect to receive, workers’ compensation benefits? {radio}

Yes

No 
- [ ] The user is able to select help text
- [ ] The user is able to move to the next step in the process, Story#14537: Form 21-8940: Ability to populate Impact on military duty interview questions
- [ ] Field validations (if applicable per the linked document for Validations above) are enforced in the UI
- [ ] Error messages (if applicable per the linked document for Validations above) are displayed in the UI
**NOTE: No fields are required as per document.**

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by desireeturner** [comment]
This PR is related to the following issue: https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/14536

**Review by aaronjlech — CHANGES_REQUESTED** [review]
@imeecuison Looks good, just some minor little things!

File: src/applications/disability-benefits/all-claims/pages/supplementalBenefits.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (imeecuison) [reply src/applications/disability-benefits/all-claims/pages/supplementalBenefits.js]: changed!

File: src/applications/disability-benefits/all-claims/pages/supplementalBenefits.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+);
+
+export const uiSchema = {
+  'ui:title': unemployabilityTitle,
+  'ui:description': supplementalBenefitsTitle,
+  receiveExpectDisabilityRetirement: {
+    'ui:title':
+      'Do you receive, or expect to receive, disability retirement benefits?',
+    'ui:widget': 'yesNo',
+    'ui:options': {},
```
[the remark this answers is not in the document]
Comment (imeecuison) [reply src/applications/disability-benefits/all-claims/pages/supplementalBenefits.js]: removed

File: src/applications/disability-benefits/all-claims/pages/supplementalBenefits.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  },
+  receiveExpectWorkersCompensation: {
+    'ui:title':
+      'Do you receive, or expect to receive, workers‘ compensation benefits? ',
+    'ui:widget': 'yesNo',
+    'ui:options': {},
```
[the remark this answers is not in the document]
Comment (imeecuison) [reply src/applications/disability-benefits/all-claims/pages/supplementalBenefits.js]: removed

File: src/applications/disability-benefits/all-claims/tests/pages/supplementalBenefits.unit.spec.jsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        schema={schema}
+        uiSchema={uiSchema}
+        data={{}}
+        formData={{}}
+        onSubmit={onSubmit}
+      />,
+    );
+
+    form.find('form').simulate('submit');
+    expect(form.find('.usa-input-error-message').length).to.equal(0);
```
[the remark this answers is not in the document]
Comment (imeecuison) [reply src/applications/disability-benefits/all-claims/tests/pages/supplementalBenefits.unit.spec.jsx]: changed

**Review by aaronjlech — APPROVED** [review]

**Comment by imeecuison** [comment]
@department-of-veterans-affairs/code-reviewers-for-external-teams-group Could I get a code review on this at your earliest convenience. Thank you!

**Review by rianfowler — APPROVED** [review]
LGTM

File: src/applications/disability-benefits/all-claims/pages/supplementalBenefits.jsx [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    receiveExpectDisabilityRetirement: {
+      'ui:title':
+        'Do you receive, or expect to receive, disability retirement benefits?',
+      'ui:widget': 'yesNo',
+    },
+    receiveExpectWorkersCompensation: {
+      'ui:title':
+        'Do you receive, or expect to receive, workers‘ compensation benefits? ',
+      'ui:widget': 'yesNo',
+      'ui:options': {},
```
Comment (aaronjlech) [inline src/applications/disability-benefits/all-claims/pages/supplementalBenefits.jsx]: This looks like it's still in the file, can we remove this empty `ui:options`?

**Review by aaronjlech — CHANGES_REQUESTED** [review]

File: src/applications/disability-benefits/all-claims/pages/supplementalBenefits.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (imeecuison) [reply src/applications/disability-benefits/all-claims/pages/supplementalBenefits.js]: sorry about that! i thought I did it.

**Review by aaronjlech — APPROVED** [review]

**Pull request merged**